web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1329,9 +1329,25 @@ $.extend(fixmystreet.set_up, {
     };
 
     var focusFirstVisibleInput = function() {
-        // Ignore logged-in form here, because it should all be pre-filled already!
-        $('#form_sign_in_yes input, #form_sign_in_no input').filter(':visible').eq(0).trigger('focus');
-    };
+        // Check if the .js-new-report-user-shown section is visible
+        var $newReportUserShown = $('.js-new-report-user-shown');
+        if ($newReportUserShown.is(':visible')) {
+          // Check if there is an element with class .form-section-preview within the .js-new-report-user-shown section
+          var $formSectionPreview = $newReportUserShown.find('.form-section-preview').first();
+          if ($formSectionPreview.length) {
+            $formSectionPreview.children().first().trigger('focus');
+          } else {
+            // Ignore logged-in form here, because it should all be pre-filled already!
+            var $inputs = $newReportUserShown.find('#form_sign_in_yes input, #form_sign_in_no input').filter(':visible');
+            $inputs.eq(0).trigger('focus');
+          }
+        }
```

Comment:
This also lost the else for all the other times this function is called (also fixed in fixup)